app/cdap/components/RecipeDetails/DetailContainer/index.tsx

```diff
@@ -0,0 +1,175 @@
+    name: string;
+    generation: number;
+  };
+  recipeId: string;
+}
+
+interface IRecipeItem {
+  recipeId: IRecipeId;
+  recipeName: string;
+  description: string;
+  directives: string[];
+  createdTimeMillis: number;
+  updatedTimeMillis: number;
+  recipeStepsCount: number;
+}
+
+interface IRecipeDetailsProps {
+  recipeDetails: IRecipeItem;
+}
+
+const PREFIX = 'features.WranglerNewUI.RecipeDetails';
+
+const RecipeDetailWrapper = styled(Box)`
+  margin-top: 19px;
+`;
+
+const RecipeName = styled(Typography)`
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 150%;
+  color: ${grey[700]};
+  text-transform: capitalize;
+`;
+
+const StepDetail = styled(Box)`
+  margin: 10px 0;
+`;
+
+const DescriptionDetail = styled(Box)`
+  margin: 20px 0;
+`;
+
+const RecipeDetailText = styled(Typography)`
+  color: ${grey[700]};
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 150%;
+  display: flex;
+  align-items: center;
+`;
+
+const RecipeStepText = styled(Typography)`
+  color: ${grey[700]};
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 150%;
+  display: flex;
+  align-items: center;
+`;
+
+const StepsGridWrapper = styled(Box)`
+  display: grid;
+  grid-template-columns: 20% 80%;
+  align-items: center;
+  padding: 15px 10px;
+`;
+
+const StepsGridWrapperHead = styled(StepsGridWrapper)`
+  padding: 9px 10px;
+`;
+
+const StepsGridHead = styled(Typography)`
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 150%;
+  color: ${grey[700]};
+`;
+
+const HeadDivider = styled(Box)`
+  width: 100%;
+  border-bottom: 1px solid ${grey[700]};
+  opacity: 0.6;
+`;
+
+const CellDivider = styled(Box)`
+  width: 100%;
+  border-bottom: 1px solid ${grey[300]};
+`;
+
+const VerticalDivider = styled(Box)`
+  width: 1px;
+  height: 25px;
+  border-left: 1px solid ${grey[300]};
+  margin: 0 10px;
+`;
+
+const getSerialNumber = (recipeStepIndex: number) => {
+  if (recipeStepIndex < 10) {
+    return `0${recipeStepIndex + 1}`;
+  } else {
+    return `${recipeStepIndex + 1}`;
+  }
+};
+
+export default function({ recipeDetails }: IRecipeDetailsProps) {
+  return (
+        <RecipeDetailWrapper>
+          <RecipeName component="h5" data-testid="recipe-name">
+            {recipeDetails.recipeName}
+          </RecipeName>
+          <StepDetail>
+            <RecipeDetailText component="div" data-testid="recipe-count-and-date">
+              {`${recipeDetails.directives.length} ${T.translate(
+                `${PREFIX}.tableHeaders.recipeStep`
+              )}`}
+              <VerticalDivider /> {dateFormatting(recipeDetails.createdTimeMillis)}
+            </RecipeDetailText>
+          </StepDetail>
+          <DescriptionDetail>
+            <RecipeDetailText data-testid="recipe-decription">
+              {recipeDetails.description}
+            </RecipeDetailText>
+          </DescriptionDetail>
+          <StepsGridWrapperHead>
+            <StepsGridHead component="body1" data-testid="recipe-step-serial-number-column-head">
```

Comment:
body1 is a variant of Typography and not a component
plz update this
components are any HTML tags like div, span, p etc.

Reply:
Done